doc/source/tutorials/local/Exploring_data/Retrieving_remote_data.rst

```diff
@@ -0,0 +1,215 @@
+Retrieving data from remote archives
+====================================
+
+This tutorial covers the retrieval of data from the ICOS Carbon Portal
+and the CEDA archives.
+
+ICOS
+----
+
+It's easy to retrieve atmospheric gas measurements from the `ICOS Carbon
+Portal <https://www.icos-cp.eu/observations/carbon-portal>`__ using
+OpenGHG. To do so we'll use the ``retrieve_icos`` function from
```

Comment:
This is no longer the `openghg.client` submodule right? If so, this needs to be updated